internal/server/migrations.go

```diff
@@ -16,3 +23,15 @@ func (a *API) addRewrites() {
 // addRedirects for API endpoints that have moved to a different path
 func (a *API) addRedirects() {
 }
+
+func (a *API) deprecatedRoutes(noAuthnNoOrg *gin.RouterGroup) {
```

Comment:
This feels weird since there would be no way to add deprecated routes to other RouterGroups.

Reply:
yeah, you'd have to pass in the other router group as an argument. I could pass in the base router group and let you redefine your own middleware etc, but that felt weird too